ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/logging.cfg

```diff
@@ -1,12 +1,14 @@
+!
+logging event storm-control discards global
+logging event storm-control discards interval 10
+!
 logging event congestion-drops interval 10
 !
 logging repeat-messages
 logging buffered 1000000 warnings
 no logging trap
 logging console errors
 no logging event link-status global
```

Comment:
@laxmikantchintakindi , can you check that this new command is in the right place? during review it felt like it was but now I have a doubt looking at my output

```
!
logging repeat-messages
logging console warnings
logging vrf mgt source-interface Management0
!
logging level AAA warnings
logging level ACL critical
!
no logging event link-status global
!
```

Just want to make sure it is not only my device

Reply:
The order is correct.

Reply:
You are right it is not in right order we need to move `no logging event link-status global` underneath  
```
logging level AAA warnings
logging level ACL critical
logging level BGP 0
```

Reply:
I adjusted the eos j2 template for correct sequence.